js/src/3rdparty/etherscan/call.js

```diff
@@ -23,19 +25,14 @@ const options = {
 
 export function call (module, action, _params, test) {
   const host = test ? 'testnet.etherscan.io' : 'api.etherscan.io';
-  let params = '';
-
-  if (_params) {
-    Object.keys(_params).map((param) => {
-      const value = _params[param];
 
-      params = `${params}&${param}=${value}`;
-    });
-  }
+  const query = stringify(Object.assign({
```

Comment:
Yeah, I also thought about this and in doubt, I'm always for the more lightweight/simpler solution. But in this case, things will actually break with perfectly valid api calls.